src/xdainterface.cpp

```diff
@@ -183,6 +183,32 @@ void XdaInterface::registerPublishers()
     }
 }
 
+void XdaInterface::registerSubscribers(){
+    //Initialise GPS_Fix(NavSatFix) Subscriber
+    std::string fix_topic_name; 
+    ros::param::get("/xsens_mti_node/fix_topic_name", fix_topic_name);
+
+    ros::param::get("/xsens_mti_node/use_fix_topic_to_set_lla", use_fix_topic_to_set_lla);
```

Comment:
Same here, suggest to remove prefix, params don't normally need the node name because node name and namespace will be attached in launch files.
Suggested a more intuitive param name:  set_lla_with_gps_fix